modules/age.nix

```diff
@@ -43,7 +43,7 @@ let
         type = types.str;
         default = config._module.args.name;
         description = ''
-          Name of the file used in /run/secrets
+          Name of the file used in /run/agenix
```

Comment:
This will need to be called out in the README or something. Will do after the actual implementation is in a good place.